packages/components/src/button/index.js

```diff
@@ -42,9 +43,13 @@ export function Button( props, ref ) {
 	const tag = href !== undefined && ! disabled ? 'a' : 'button';
 	const tagProps = tag === 'a' ? { href, target } : { type: 'button', disabled };
 
+	const passThruProps = additionalProps.instanceId ?
+		omit( additionalProps, [ 'instanceId' ] ) :
```

Comment:
This assumes this element explicitly uses the `instanceId` prop for me. I think this prop shouldn't be passed at all instead.

Reply:
Yup agreed, and the culprit was `<MenuItem/>`